app/views/users/show.html.erb

```diff
@@ -29,7 +32,7 @@
         
   </div>
 
-  <form action="./<%= @user.id%>/posts" method="get" class="see-all-posts">
+  <form action="./<%= @user.id%>/posts" method="get">
     <button type="submit" class='btn'>See all posts</button>
   </form>
```

Comment:
- [ ] [OPTIONAL] You may consider fixing this button, this button should show all the posts for a user, not just the most recent ones.